ServerCore/Pages/FilesController.cs

```diff
@@ -44,9 +44,10 @@ public async Task<IActionResult> Index(string eventId, string filename)
             // * Admins have access to all files in their event
             // * Authors have access to all files attached to puzzles they own
             // * Players can see puzzles and materials on puzzles they've unlocked
+            // * Players can see puzzles and materials in alpha-testing events when the puzzle needs alpha tests
             // * Players can see answers after the event's AnswersAvailable time
             // * Players can see solve tokens on puzzles they've solved
-            if (!await IsAuthorized(eventObj.ID, content.Puzzle, content))
+            if (!(eventObj.IsAlphaTestingEvent && content.Puzzle.AlphaTestsNeeded > 0) && !await IsAuthorized(eventObj.ID, content.Puzzle, content))
```

Comment:
Wouldn't this addition be better in the IsAuthorized method?

Reply:
I'll have to change the first argument of IsAuthorized from an ID to an Event, but sure that makes sense and the resulting clarity is worth the slightly larger diff.